src/test/java/uk/gov/companieshouse/exemptions/util/LocalDateDeSerializerTest.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.companieshouse.exemptions.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.time.LocalDate;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.function.Executable;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.exemptions.exception.BadRequestException;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
No mocks are used so don't need the MockitoExtension or `@BeforeEach` section. Instantiate class under test inline.